### DISCUSSION TITLE
Using alternate pin number for USART

### DISCUSSION THREAD

**Discussion by mayurharge** [SpenceKonde/megaTinyCore#897] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42429307/212913350-d1c2f4e3-0334-4fed-89f9-c8d1ed884d5d.png)
this is the snippet for datasheet of attiny1607,
as per datasheet we can use PA1 and PA2 as hardware serial0 pins
in file pins_arduino.h there is provision for pin swapping PIN_HWSERIAL0_TX_PINSWAP_1
I am not able to understand how to enable these pins as my primary serial 
please let me know how to achieve this

**Comment by hmeijdam** [comment, marked as answer]
Serial.swap();// use this to set the TXD*/RXD* to their alternate pins

**Reply by mayurharge** [reply]
Thanks 
Its working. used Serial.swap(1);

**Comment by SpenceKonde** [comment]
Serial.swap(1) see Ref.serial

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***
